include/seahorn/Transforms/Utils/DevirtFunctions.hh

```diff
@@ -22,6 +22,8 @@ class CompleteCallGraphAnalysis;
 
 namespace seahorn {
 
+//#define USE_BOUNCE_FUNCTIONS
```

Comment:
you can go all the way and add a cmake option to turn this on during compilation

Reply:
Yes